packages/ui/src/components/va-affix/VaAffix.vue

```diff
@@ -18,6 +18,7 @@
 <script lang="ts">
```

Comment:
https://github.com/epicmaxco/vuestic-ui/pull/1634#issuecomment-1096474275

Fallback to white instead of `default` value doesn't seem like correct behavior.

Reply:
4 digit CSS color is RGBA, so in `#ff00` the last 0 is alpha.

`#ff00` is the same as `#ffff0000`.

I was wrong about white color, it is just transparent color.

Try `#ff09`

Reply:
It was transparent because "0" is getting parsed as alpha in this color `#ff00`.

Should be fine. Though we could throw a warning instead of the error, but this is not that important.

Reply:
We throw it only for 5 digit values.